cartography/intel/azure/function_app.py

```diff
@@ -134,6 +135,7 @@ def _load_function_apps_tx(
     f.type = function_app.type,
     f.location = function_app.location,
     f.region = function_app.location,
+    f.hostNamesDisabled = function_app.hostNamesDisabled,
```

Comment:
does enabling host name makes the functions public?

Reply:
yes